Update newsListFetchItems.md (#1350)
contao/docs: docs/dev/reference/hooks/newsListCountItems.md

Return `false` if this hook should not be considered. Return an integer otherwise.

> After an integer has been returned, no further hooks of type `newsListFetchItems` will be executed!

Reviewer: ```suggestion
If the return value is other than `false`, no further hooks of type `newsListCountItems` will be executed!
```

I think we should keep the same phrasing (as the check is `=== false`).
